fastlane/Fastfile

```diff
@@ -1,27 +1,207 @@
+        options[:build_type] = "RC"
+        options[:override_folder] = "COLUMN3-CI-configuration"
+        options[:s3_subfolder] = "custom/bund/column3"
+        options[:app_center_app_name] = "Wire-by-Bund-Column3"
+        options[:upload_dsyms_to_app_center] = false
+        options[:run_security_tests] = true
+        build_and_upload_app_store(options)
+    end
+
+    desc "Build & Upload Col 1 Production AppStore build"
+    lane :appstore_col_1_prod do |options|
+        options[:build_type] = "AppStore"
+        options[:override_folder] = "BK-CI-configuration"
+        options[:s3_subfolder] = "custom/bund/c1-appstore"
+        options[:app_center_app_name] = "Wire-by-Bund-C1-App-Store"
+        options[:upload_dsyms_to_app_center] = false
+        options[:run_security_tests] = true
+        build_and_upload_app_store(options)
+    end
+
+    desc "Build & Upload Col 3 Production AppStore build"
+    lane :appstore_col_3_prod do |options|
+        options[:build_type] = "AppStore"
+        options[:override_folder] = "COLUMN3-CI-configuration"
+        options[:s3_subfolder] = "custom/bund/c3-appstore"
+        options[:app_center_app_name] = "Wire-by-Bund-C3-App-Store"
+        options[:upload_dsyms_to_app_center] = false
+        options[:run_security_tests] = true
+        build_and_upload_app_store(options)
+    end
+
+    def build_and_upload_app_store(options)
+        options[:build_number] = generate_build_number
+        prepare_for_release(options)
+        if options[:run_security_tests]
+            security_plans(options)
+        end
+        build_for_release(options)
+        upload_app_store(options)
+        upload_s3(options)
+        if options[:upload_dsyms_to_app_center]
+            upload_dsyms_to_app_center_for_appstore_build(options)
+        end
+    end
+
+    def fetch_latest_configuration_repo 
+        path = '../wire-ios-build-assets'
+        `rm -rf #{path}`
+        if is_running_on_ci 
+            Git.clone(
+                "https://#{ENV['WIRE_INTERNAL_GITHUB_USER']}:#{ENV['WIRE_INTERNAL_GITHUB_TOKEN']}@github.com/wireapp/wire-ios-build-assets.git",
+                path
+            )
+        else
+            Git.clone(
+                "git@github.com:wireapp/wire-ios-build-assets.git",
+                path
+            )
+        end
+
+    end
+
+    def download_distribution_certificate_and_profiles(build)
+        if is_running_on_ci 
+            match(
+                git_url: "https://#{ENV['WIRE_INTERNAL_GITHUB_USER']}:#{ENV['WIRE_INTERNAL_GITHUB_TOKEN']}@github.com/wireapp/wire-ios-certificates.git",
+                readonly: true,
+                type: "appstore",
+                skip_provisioning_profiles: true,
+                app_identifier: []
+            )
+        else
+            match(
+                readonly: true,
+                type: "appstore",
+                skip_provisioning_profiles: true,
+                app_identifier: []
```

Comment:
should we put the app_bundle_id from below here?

Reply:
no because certificates are not relying on app ids and provisioning profiles is skipped here